contrib/buildsystems/engine.pl

```diff
@@ -40,6 +41,7 @@ sub showUsage
 # Parse command-line options
 while (@ARGV) {
     my $arg = shift @ARGV;
+	#print "Arg: $arg \n";
```

Comment:
>In contrib/buildsystems/engine.pl:
>> @@ -40,6 +41,7 @@ sub showUsage
>>  # Parse command-line options
>>  while (@ARGV) {
>>      my $arg = shift @ARGV;
>> + #print "Arg: $arg \n";

>I think there was some discussion about adding dead code on the mailing 
>list...

The point was that it isn't 'dead code', rather that it's "deactivated 
code".
http://article.gmane.org/gmane.comp.version-control.msysgit/21710

> I would actually agree that it would be better to avoid adding it.

To me, it was a carefully chosen indicator of where to activate code 
when an upstream change causes the code to fail. This code matches the 
existing code style present in a few key places which helped me in 
trying to understand the convoluted code stack.

I had already removed a large number of debug statements which I'd used 
at various times.

I'm still of the opinion that it is worth having for those that follow 
later. (and the limited other places)
--
Philip